aws/rust-runtime/aws-config/src/profile/credentials/exec.rs

```diff
@@ -100,6 +101,9 @@ impl ProviderChain {
                     })?
             }
             BaseProvider::AccessKey(key) => Arc::new(key.clone()),
+            BaseProvider::CredentialProcess(credential_process) => Arc::new(
+                CredentialProcessProvider::new(credential_process.to_string()),
```

Comment:
if `new` returned a result, we could return a profile file error here which is probably the right thing